### ISSUE TITLE
Better locale

### ISSUE THREAD

**Issue by ImUrX** [Fabulously-Optimized/fabulously-optimized#838]
### CurseForge link

https://www.curseforge.com/minecraft/mc-mods/better-locale

### CurseForge Mod Distribution

Allowed

### Modrinth link

https://modrinth.com/mod/better-locale

### Source/other link

https://github.com/ImUrX/localizedbrowser

### Mod file size

346 KB

### License

(any other license)

### What it does

It simplifies the search of items in creative/recipe book by not requiring the usage of items with graphemes (i.e. `á`, `ö`)

### Why should it be in the modpack

Just makes Minecraft more accessible in other languages, it also contains specific utilities for Japanese as an hiragana/katakana IME and a kanji dictionary for being able to search stuff just in hiragana

### Why shouldn't it be in the modpack

* Downloads a 40MB dictionary when using Japanese
* Requires Kotlin

**Comment by Madis0** [comment]
Hmm. Currently the Language Reload mod already allows you to search in as many languages as you want, which might solve this use case for some.

**Comment by ImUrX** [comment]
> What do you mean by "denied for some reason"? There is always a reason given, or it might just be in longer review.

![image](https://github.com/Fabulously-Optimized/fabulously-optimized/assets/3698237/6a0d16a4-ed39-4b07-b9d5-e3f67f07b73f)
which is the Japanese Kotlin library I use is done jar-in-jar in fabric and in Neoforge it's done via shadow. I don't know why they think it's obfuscated in fabric and not in Neoforge :/
I'm waiting for Curseforge response on more info

> Hmm. Currently the Language Reload mod already allows you to search in as many languages as you want, which might solve this use case for some.

Language reload indeed solves the issue if you know english, that's not true for everyone.

**Comment by ImUrX** [comment]
its now in curseforge too

**Comment by Madis0** [comment]
> Language reload indeed solves the issue if you know english, that's not true for everyone.

That is true. However, as a person who has õäöüšž in my native language, I've never felt like myself or others I know actually struggle with this... 
At the same time, I can understand that languages with letters like àáâāãą, which may not be explicitly on the keyboard and can be in the middle of a lot of words, could struggle. Then it is a matter of finding out how many of those words exist in Minecraft 🤔

> it also contains specific utilities for Japanese as an hiragana/katakana IME and a kanji dictionary for being able to search stuff just in hiragana

How does this benefit the Japanese? Why would they use this mod instead of something like #602?

**Comment by ImUrX** [comment]
> How does this benefit the Japanese? Why would they use this mod instead of something like https://github.com/Fabulously-Optimized/fabulously-optimized/issues/602?

The mod you mention what it does is block IME input that is permanent in Minecraft as currently there is no way to use it just when writing because GLFW doesn't support such a thing currently.
This let's you use an english keyboard as input and it's better coupled with Minecraft itself.

> Then it is a matter of finding out how many of those words exist in Minecraft 🤔

In spanish a lot of people just don't use tildes or dieresis as it's too much trouble (that includes me)
